docker/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN yum -y install --setopt=skip_missing_names_on_install=False \
                    rh-ruby25-rubygem-net-telnet devtoolset-9-gcc-c++ git && \
     source /opt/rh/rh-ruby25/enable && \
     source /opt/rh/devtoolset-9/enable && \
-    gem install libxml-ruby gnuplot distribution test-unit builder concurrent-ruby && \
+    gem install libxml-ruby gnuplot distribution test-unit builder concurrent-ruby facter && \
```

Comment:
This also needs to be put into the dev image:
https://github.com/vespa-engine/docker-image-dev/blob/master/dev/centos7/Dockerfile

Alternatively read the file /etc/redhat-release and look for CentOS

Reply:
Changed to read `/etc/redhat-release`